README.md

```diff
@@ -208,6 +210,8 @@ prepare some data, and then fork out to several parallel tasks that use the disk
        height="213"/>
 </p>
 
+## GCE Persistent Disk
+
```

Comment:
Can we add a link to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#gce to make it a little clearer on how to make the storage class (and what they are).